### ISSUE TITLE
Downloader for mobile leaves old version in place and saving issues

### ISSUE THREAD

**Issue by GetOutOfMyLab** [renpy/renpy#5473]
I posted about this on the forums: https://lemmasoft.renai.us/forums/viewtopic.php?p=566760#p566760

I've spent almost 2 hours trying different things just to see if I could get it to work. For my first attempt, I followed the [guide](https://www.renpy.org/doc/html/downloader.html).

The downloader game has it's own package name: com.domain.game-dl, for example. I install it on my Android, run it, it downloads files from the server, launches the main game and I'm good to go. Or am I? The saves from the previous install are not showing and if I start a new game, I'm unable to save. Presumably this is because the guide instructs us to disable saving: 
```ren'py
define config.save = False
``` 

Also, the previous installation of the game is still there. So I thought, maybe I need to use the same package name and update it. So I go to the downloader Android configuration, change the package name to: com.domain.game, for example. Install that on my Android and now it actually updates the previous game, but then all saves are lost and saving still does not work. 

I figured this part would be much more difficult than setting up the server to support this, but I guess I was wrong. 

Maybe there's something missing from the guide or I'm doing something stupid, but I can't see how to get this working. 

Ren'Py 7.7.1

**Comment by GetOutOfMyLab** [comment]
I'm cautiously optimistic that I got this working. Seems that in the main game, I had to explicitly set define config.save = True (since the Downloader app sets it to False). 

I also made sure the package name was the same so that it would update the old version of the app. Saves appear to be loadable and saveable now. 

Not sure if this is the "right" answer, but if it is, it would help to have specifics outlines in the documentation.

**Comment by renpytom** [comment]
You shouldn't have to set it to true - just not set it to False. There shouldn't be anything shared between the downloader and main games.

**Comment by GetOutOfMyLab** [comment]
> You shouldn't have to set it to true - just not set it to False. There shouldn't be anything shared between the downloader and main games.

Then here are my questions:

1. If I have an older packaged android version installed before I implemented the downloader, should there be two separate apps after I install my downloader? What if I packaged up a full android version that is up-to-date: would there still be the main game and downloader showing as separate apps on the phone?
2. When I installed the downloader without any previous app on the phone, it would launch the main game, but I was still unable to save. I assumed because the downloader disabled them, but sounds like this shouldn't happen, correct?
3. When you say there should be nothing shared, does that mean that saves from previous versions won't be recoverable? Or are you just referring to package name (e.g. com.mycompany.mydownloader), save location, etc.? 

Thanks
